docs/DeveloperGuide.md

```diff
@@ -170,6 +170,33 @@ sequence diagram shows the interactions within `Model` when editing a person.
 
 ![EditSequenceDiagram-Model](images/EditSequenceDiagram-Model.png)
 
+### Delete Tag Command
+
+This section aims to show the logic behind delete tag command and the consideration behind the scene.
+
+Delete Tag Command is a new featured added for user to delete certain tag associated with a specific person located by index of the current list. This command is called by `deletetag` followed by the index of the targeted person, then a target tag `tag` object. In case of the targeting tag is not associated with the person, an error message suggesting missing targeted tag will be returned.
+
```

Comment:
I think it is important to specify that you can only delete 1 tag? This is because add, edit and addtag each allows for multiple tags. I think it is important to state such specification and possibly the design choice? Or maybe add deletion of multiple tags for a future implementation.